### PR TITLE
Rewrite CI to use GitHub actions

### PR DESCRIPTION
**Pull request by Shadowghost**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    uses: Shadowghost/jellyfin-docs/.github/workflows/_meta.yaml@ci-rework
```
Comment (1337joe) [inline .github/workflows/build.yaml:14]: Shouldn't this (and publish.yaml) point to `jellyfin/jellyfin-docs` instead of `Shadowghost/jellyfin-docs`?

Comment (Shadowghost) [reply .github/workflows/build.yaml:14]: Yes but to test it I need to reference an existing workflow :)

File: .github/renovate.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            "automerge": true,
+            "automergeType": "branch"
+        }
+    ],
+    "git-submodules": {
+        "enabled": true
+    },
+    "timezone": "Etc/UTC",
+    "schedule": ["before 2am"]
+}
```
Comment (nielsvanvelzen) [inline .github/renovate.json:22]: Missing a new line

**Review by nielsvanvelzen — APPROVED** [review]

File: .github/renovate.json (hunk 2 above)
Comment (Shadowghost) [reply .github/renovate.json:22]: VS Code autmatically removes newlines at the end of json files.

Is it really required?

Comment (nielsvanvelzen) [reply .github/renovate.json:22]: We generally have those for all files enabled via the editorconfigs. I think this repository does too.

Comment (Shadowghost) [reply .github/renovate.json]: You are correct. The weird thing is I have the editorconfig plugin installed on vscode but it seem to either be ignored or overridden by another setting.

Nonetheless I fixed it manually.

**Comment by crobibero** [comment]
Should the `.ci` folder be removed so re disable azure pipelines?

**Comment by Shadowghost** [comment]
Oh right, I forgot that. I'll add a commit.

**Review by crobibero — APPROVED** [review]

**Pull request merged**